_SQLServer/TableLoadTreasure.sql

```diff
@@ -0,0 +1,408 @@
+'http://htkb.info/SDE/1.0/Cards/Treasure/Specific/Card_Treasure_Fire_Gel_Goo_1.0.png', 
+'http://htkb.info/SDE/1.0/Cards/Treasure/Specific/Card_Loot_1.0_back.png', --? Is there no back?
+'http://htkb.online/SDWiki/SDE/1.0/Cards/Treasure/Specific/Fire_Gel_Goo.xhtml', 
+'Treasure', '1.0', '1.0 Core Set', 'Default',
+'Oh gross!');
+insert into Equipment (Position, Effect) VALUES ('Top', '+1G WILL'); --? Not sure that this should be "Equipment" and the values need to be checked - everything about this line needs to be checked - Card had Splash on it - should this information go somewhere?
+
+
+--Fool's Gold
+insert into Cards (Name, PictureFront, PictureBack, Link, CardType, ProductSet, Module, Mode, Flavor) VALUES ('Fool''s Gold',
+'http://htkb.info/SDE/1.0/Cards/Treasure/Specific/Treasure_Fools_Gold_1.0.png', 
+'http://htkb.info/SDE/1.0/Cards/Treasure/Specific/Card_Loot_1.0_back.png', --? Is there no back?
+'http://htkb.online/SDWiki/SDE/1.0/Cards/Treasure/Specific/Fools_Gold.xhtml', 
+'Treasure', '1.0', '1.0 Core Set', 'Default',
+'Better lucky than smart, I always say.');
+insert into Equipment (Position, Effect) VALUES ('Top', 'At then end of each of this model''s activations draw one loot card.  Anytime this model makes a roll using its WILL attribute discard the highest result rolled.'); --? Not sure that this should be "Equipment" and the values need to be checked - everything about this line needs to be checked - 
+
+
+--Gem of Greed
+insert into Cards (Name, PictureFront, PictureBack, Link, CardType, ProductSet, Module, Mode, Flavor) VALUES ('Gem of Greed',
+'http://htkb.info/SDE/1.0/Cards/Treasure/Specific/Card_Treasure_Gem_Of_Greed_1.0.png', 
+'http://htkb.info/SDE/1.0/Cards/Treasure/Specific/Card_Loot_1.0_back.png', --? Is there no back?
+'http://htkb.online/SDWiki/SDE/1.0/Cards/Treasure/Specific/Gem_Of_Greed.xhtml', 
+'Treasure', '1.0', '1.0 Core Set', 'Default',
+'Mine, it''s all mine!');
+insert into Equipment (Position, Effect) VALUES ('Top', 'A Hero with the Gem of Greed may equip two loot cards to every equipment slot except relics.'); --? Not sure that this should be "Equipment" and the values need to be checked - everything about this line needs to be checked - 
+
+
+--Grimy Grim Granite Greaves
+insert into Cards (Name, PictureFront, PictureBack, Link, CardType, ProductSet, Module, Mode, Flavor) VALUES ('Grimy Grim Granite Greaves',
+'http://htkb.info/SDE/1.0/Cards/Treasure/Specific/Card_Treasure_Grimy_Grim_Granite_Greaves_1.0.png', 
+'http://htkb.info/SDE/1.0/Cards/Treasure/Specific/Card_Loot_1.0_back.png', --? Is there no back?
+'http://htkb.online/SDWiki/SDE/1.0/Cards/Treasure/Specific/Grimy_Grim_Granite_Greaves.xhtml', 
+'Treasure', '1.0', '1.0 Core Set', 'Default',
+'Three times strong!');
+insert into Equipment (Position, Effect) VALUES ('Top', '+1R Armor'); --? Not sure that this should be "Equipment" and the values need to be checked - everything about this line needs to be checked - 
+
```

Comment:
Keyword assignment line is missing for "Immune: Knockdown".
Keyword assignment line is missing for "Immune: Slow".
Keyword assignment line is missing for "Immune: Immobile".